src/options.js

```diff
@@ -1,50 +1,103 @@
 'use strict';
 
 const _ = require('lodash');
-const getContent = require('./getContent');
-
-const displayUrl = document.getElementById('display_url');
-const editUrl = document.getElementById('edit_url');
-const url = document.getElementById('template_url');
-const urlInput = document.getElementById('url_input');
-const autoFill = document.getElementById('auto_fill');
-
-function showEdit(){
-  displayUrl.style.display = 'none';
-  editUrl.style.display = 'block';
-}
+const React = require('react');
 
-function submitUrl(){
-  const newUrl = urlInput.value;
-  chrome.storage.sync.set({ templateUrl: newUrl });
-  getContent();
-  url.innerHTML = newUrl;
-  url.href = newUrl;
-  editUrl.style.display = 'none';
-  displayUrl.style.display = 'block';
-}
+const getContent = require('./getContent');
+const style = require('../style/options.js');
 
-function toggleAutoFill(){
-  chrome.storage.sync.set({ autoFill: autoFill.checked });
-  if(autoFill.checked){
-    chrome.tabs.query({ url: 'https://github.com/*/*' }, function(tabs){
-      _.forEach(tabs, function(tab){
-        chrome.tabs.sendMessage(tab.id, { fillPR: true });
-      });
+const EditUrl = React.createClass({
+  getInitialState: function(){
+    return {
+      templateUrl: '',
+      deltaUrl: '',
+      displayUrl: 'block',
+      editUrl: 'none'
+    };
+  },
+  componentDidMount: function(){
+    chrome.storage.sync.get('templateUrl', (res)=>{
+      if(this.isMounted()){
+        this.setState({
+          templateUrl: res.templateUrl,
+          deltaUrl: res.templateUrl
+        });
+      }
     });
+  },
+  handleUrlChange: function(){
+    this.setState({ deltaUrl: event.target.value });
+  },
+  handleEdit: function(){
+    this.setState({ displayUrl: 'none' });
+    this.setState({ editUrl: 'block' });
+  },
+  handleSubmit: function(){
+    this.setState({ displayUrl: 'block' });
```

Comment:
setState can take an object with multiple keys, no reason to call it multiple times